src/index.html

```diff
@@ -37,8 +37,7 @@
 									target="_blank">Who's On First</a>
 							</li>
 							<li class="nav-item">
-								<a class="nav-link" href="https://bombmanual.com/web/index.html#Memory"
-									target="_blank">Memory</a>
+								<a class="nav-link" href="#Memory">Memory</a>
```

Comment:
```suggestion
								<a class="nav-link" href="#Memory" data-href="https://bombmanual.com/web/index.html#Memory">Memory</a>
```
That allows us to easily add the manual link later without having to go and look up the url again.